docs/change-controlled-docs.rst

```diff
@@ -0,0 +1,233 @@
+5. The release manager creates a new DocuShare tag (``docushare-vN``) from the head of the release branch, uploads the revised document to DocuShare, and notifies the CCB.
+
+.. figure:: ccd-amend.svg
+   :name: fig-amend
+   :align: center
+   :target: ccd-amend.svg
+   :alt: Git commit tree showing amendments and resubmission to a CCB.
+
+   Example Git tree showing how CCB feedback is addressed on a release branch.
+   Two individuals address CCB feedback by creating separate ticket branches (``tickets/DM-3`` and ``tickets/DM-4``) from the release branch (``RFC-1``) that are merged back to the ``RFC-1`` release branch
+   The release manager resubmits the amended document to the CCB by creating a docushare tag (``docushare-v2``) and uploading version 2 of the document to DocuShare.
+
+.. _ccd-release:
+
+Releasing a new baselined version of the document
+=================================================
+
+When the CCB baselines (accepts) the new document, the project librarian designates the baselined document with a semantic version.
+The DM release manager is responsible for releasing a document.
+
+1. From the head of the release branch (``tickets/RFC-N`` or ``tickets/LCR-N``), add a commit that:
+
+	- Updates the semantic version of the document to the one designated by the project librarian.
+	- Updates the change history table to reflect the new version.
+	- Fixes the document's date to the date of baselining.
+	- Removes the "draft" watermark (for example, by removing ``draft`` from the document class's arguments).
+
+2. Create a new DocuShare tag (``docushare-vN``) and upload the document to DocuShare.
+
+3. Once the project librarian has set that DocuShare version as the preferred version, create a semantic version tag (formatted ``v<major>.<minor>``) at the same commit as the corresponding DocuShare tag.
+
+4. Create a new ticket branch from the **head** of the ``master`` branch.
+   Cherry pick amendment commits from the release branch onto that ticket branch.
+   Also include a commit that updates the document change history table.
+   Merge that ticket branch to ``master``.
+
+**Notes:**
+
+- The release branch is never merged back to ``master``.
+  Amendments get back to ``master`` through cherry picking commits from the release branch.
+- Development is allowed to happen on the ``master`` branch while CCB review is simultaneously happening on a release branch.
+  This means that the DM release manager is responsible for properly addressing conflicts while cherry picking amendments back to ``master``.
+
+.. figure:: ccd-release.svg
+   :name: fig-release
+   :align: center
+   :target: ccd-release.svg
+   :alt: Git commit tree showing a document release.
+
+   Example Git tree showing how a document is released and baselined.
+   The release manager creates a ticket branch (``tickets/DM-6``) with two commits from the current release branch (``tickets/RFC-1``).
```

Comment:
Do we really need to create a JIRA ticket for updating the change record? I'd generally put that directly on the release branch.